src/KafkaFlow.UnitTests/Consumer/ConsumerManagerTests.cs

```diff
@@ -104,6 +104,7 @@ public void OnPartitionsAssigned_StartWorkerPool()
             // Arrange
             IConsumer<byte[], byte[]> consumer = null;
             var partitions = this.fixture.Create<List<TopicPartition>>();
+            IDependencyResolver dependencyResolver = Mock.Of<IDependencyResolver>();
```

Comment:
Please, create an attribute `IDependencyResolver dependencyResolver ` so it will not be necessary to create this variable in each test :)

Reply:
Done! :)